src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,43 @@
-import React from 'react';
-import { Loader } from '../Loader';
+import React, { useState, useEffect } from 'react';
+import { getUser } from '../../api';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { Loader } from '../Loader/Loader';
+
+type Props = {
+  todos: Todo[];
+  setOpen: React.Dispatch<React.SetStateAction<boolean>>;
+  setIsSelect: React.Dispatch<React.SetStateAction<number>>;
+  isSelect: number;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  todos,
+  setOpen,
+  setIsSelect,
+  isSelect,
+}) => {
+  const select = todos.find((todo) => todo.id === isSelect);
```

Comment:
check variable name, it's not clear why todo is stored in select variable - select is an action